cmd/medsky/events/cbor_gen.go

```diff
@@ -0,0 +1,303 @@
+// Code generated by github.com/whyrusleeping/cbor-gen. DO NOT EDIT.
```

Comment:
should probably update `indigo;gen/main.go` to keep this file up-to-date? especially as we have some tweaks and deprecations coming soon.

eg, we'll be nuking the `#handle`, `#migrate`, and `#tombstone` messages; and adding `#sync`

Reply:
This is `subscribeRepos` stuff. Ts shouldn't even be in an `events` package but next to the rest of the protocol stuff in `atproto`